requirements-docs.txt

```diff
@@ -0,0 +1,5 @@
+mkdocs==1.1.2
```

Comment:
Could we put this inside `.github/workflows` with the other requirements files? Could potentially add it to `setup.py` too, though not sure it matters much for docs building.

Reply:
I can do that if that is the consensus. I personally am against it, as I think it's obfuscating a critical part of this repository which is the current set of tested dependencies. Most projects keep `requirements-*` at the top level for visibility.

Reply:
Agree that it might be time to move all the requirements back up to the top-level, but that should probably be a separate PR.